.github/workflows/build-and_test.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Restore dependencies
         run: dotnet restore ${{ env.solutionName }}
       - name: Build
-        run: dotnet build ${{ env.solutionName }} --no-restore -c Release /p:UseSharedCompilation=false
+        run: dotnet build ${{ env.solutionName }} --no-restore -c Debug /p:UseSharedCompilation=false

```

Comment:
assuming you're switching to debug to avoid signing?

Reply:
Yes. That's the case for the PR checks.